.gitignore

```diff
@@ -0,0 +1,7 @@
+
+pycache
```

Comment:
is better to define this like so `**/__pycache__/` so that it is ignored in all subdirectories